### DISCUSSION TITLE
Brocken markup on email verification page

### DISCUSSION THREAD

**Discussion by Hydrock** [community/community#77015] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I develop my own website and i trying to find how other web site make email verification. On GitHub verification page, if you try to enter the code a lot of times you will see message: Whoa there! You have exceeded a secondary rate limit. But now this message is broken
![github-bug](https://github.com/community/community/assets/20814779/464c4415-b458-410b-bc4c-ab57bd476a8f)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Hydrock ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
